File: app/src/main/java/com/example/myapplication4/Cuota.java
package com.example.myapplication4;
import java.time.LocalDate;
public class Cuota {
    private int nroCuota;
    private LocalDate fechaPago;

    public Cuota(int nroCuota, LocalDate fechaPago){
        this.nroCuota = nroCuota;
        this.fechaPago = fechaPago;
    }
}

File: app/src/main/java/com/example/myapplication4/model/Cuota.kt
package com.example.myapplication4.model

import androidx.room.Entity
import androidx.room.ForeignKey
import androidx.room.PrimaryKey
import java.util.Date

@Entity(
    tableName = "cuota",
    foreignKeys = [ForeignKey(
        entity = Transaccion::class,
        parentColumns = arrayOf("id"),
        childColumns = arrayOf("transaccionId"),
        onDelete = ForeignKey.CASCADE
    )]
)
data class Cuota(
    @PrimaryKey(autoGenerate = true)
    val nroCuota: Int,
    val fechaPago: Date,
    val transaccionId: Int
)

